components/mediation-admin/org.wso2.carbon.mediation.library/src/main/java/org/wso2/carbon/mediation/library/service/MediationLibraryAdminService.java

```diff
@@ -148,7 +148,7 @@ public LibraryInfo getLibraryInfo(String libName, String packageName) throws Axi
 		OMElement impEl = SynapseImportSerializer.serializeImport(synImport);
 		if (impEl != null) {
 			try {
-				SynapseImport synapseImport = configuration.getSynapseImports().get("{"+packageName+"}"+libName);
+				SynapseImport synapseImport = configuration.getSynapseImports().get("{" + packageName + "}" + libName);
```

Comment:
This is still not correct, when doing code indentation, you must use spaces (4), not tabs, please fix in other places as well. This should be an IDE setting.